### ISSUE TITLE
-fcf-protection=full not supported on ppc64le

### ISSUE THREAD

**Issue by gbcox** [mariusor/mpris-scrobbler#98]
Opened a Fedora bug here: https://bugzilla.redhat.com/show_bug.cgi?id=2277631

Here is the pertinent response:
**As documented, -fcf-protection= option https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html#index-fcf-protection is only supported on i686/x86_64 targets, not on any other target (like aarch64, s390x nor ppc64le in Fedora).
So, don't use it except on i686/x86_64.** 

I did a quick look at your build and found this:

```
if build_machine.cpu() != 'aarch64'
    # this is not supported by the ARM Gcc in the Debian CI image
    add_project_arguments('-fcf-protection=full', language : 'c')
endif
```

It needs to be reworked so the fcf-protection=full is only used on i686/x86_64 targets.

**Comment by mariusor** [comment]
Pushed a new commit. Let me know if it works.

PS. Do you maintain the community build for Fedora? If that's the case: thank you. <3

**Comment by gbcox** [comment]
> Pushed a new commit. Let me know if it works.
Works!  Thanks so much for the quick response.

> PS. Do you maintain the community build for Fedora? If that's the case: thank you. <3
Yes, you're very welcome.  Thanks for developing this application!

**Comment by mariusor** [comment]
I created a new release just with this fix: [v0.5.3](https://github.com/mariusor/mpris-scrobbler/releases/tag/v0.5.3)
